### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Bootstrap#163]
@liyokuna with 1- The state "checked" should have a suficient colour contrast (non conformity :  white on orange) 2- Textual fields are grey on white solution ->use a darker grey 3- Respect the ergonomics rules about the proximity between the textual fields and its labels.  These two elements should be closer. 4- The focus on the selection lists don't have a suficient colour contrast 5- All forms with required fields should have a kind of legend at the begening indicating " required fields are etc" 6- You don't put help with data entry(aide à la saisie) as the data format (for the fields "textarea", put the number of authorized characters and for the "password" field, put the formt) to reduce the penibility when users fill in the forms. See the forms components : https://boosted.orange.com/docs/4.3/boostwatch/#forms
